### PR TITLE
reflow the stack size story

### PR DESCRIPTION
**Pull request by tshepang**

### CODE REVIEW

File: library/std/src/thread/mod.rs [hunk 1]
```diff
@@ -124,8 +124,10 @@
 //!
 //! ## Stack size
 //!
-//! The default stack size is platform-dependent and subject to change. Currently it is 2MB on all
-//! Tier-1 platforms. There are two ways to manually specify the stack size for spawned threads:
+//! The default stack size is platform-dependent and subject to change.
+//! Currently, it is 2 MiB on all Tier-1 platforms.
```
Comment (compiler-errors) [inline library/std/src/thread/mod.rs:128]: Is it `MiB` or `MB` by default?

Comment (clubby789) [reply library/std/src/thread/mod.rs:128]: 2 MiB seems correct
https://github.com/rust-lang/rust/blob/e11cb36c751550cf197e23440d53f04156c3a2a3/library/std/src/sys/unix/thread.rs#L15
https://github.com/rust-lang/rust/blob/e11cb36c751550cf197e23440d53f04156c3a2a3/library/std/src/sys/windows/thread.rs#L16

**Comment by tshepang** [comment]
friendly ping

**Pull request merged**